Resources/Private/Javascript/Components/DefinitionList.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+import { classList } from '../Helper/utilities';
```

Comment:
i just saw this, wouldnt it be better to use an npm package here? Like react-bem-helper or classnames?

Reply:
will be moved to improvements